tsconfig.json

```diff
@@ -13,10 +13,11 @@
     "noImplicitAny": true,
     "strictNullChecks": true,
     "noErrorTruncation": true,
-    "lib": ["es6", "esnext.asynciterable"]
+    "lib": ["dom", "es6", "esnext.asynciterable"]
   },
   "include": [
     "./src/**/.*.js",
+    "./src/types/**/*.d.ts",
```

Comment:
How were the `*.d.ts` files included before?

Reply:
They weren't as it wasn't required, I ported some scripts to ts and they require types